site/_scss/globals/_colors.scss

```diff
@@ -11,7 +11,7 @@ $google-colors: (
   'blue-200': #aecbfa,
   'blue-100': #d2e3fc,
   'blue-50': #e8f0fe,
-  'red-900': #a50e0e,
+  'red-900': #9f0e0e,
```

Comment:
Those are the hardcoded material colors and may not be changed. Please note that would also change all other red hues on the site.

Reply:
By looking at the example from the PR I see the change is veeeery slight. Anyway, I know that has been a problem with other PRs before. Ewa's call :)

Reply:
According to the note from Dan, the red-darkest colour var has been darkened slightly to achieve AAA contrast (#a50e0e > #9f0e0e). That change will need to be applied globally.
<img width="890" alt="Screenshot 2023-01-26 at 16 04 24" src="https://user-images.githubusercontent.com/45324326/214889888-185e09ab-8f65-45a7-8675-7001f22f166d.png">

Also, I have seen that `red-900` had been used only for `--rgb-red-darkest` var. So I decided to change the HEX code here to achieve AAA contrast with `--rgb-red-lightest`.

Reply:
I called this out specifically to Dan that our corrent palette does not fulfill accessibility requirements, with an ask to get it updated globally. So this is as expected.